resources/prosody-plugins/mod_fmuc.lua

```diff
@@ -0,0 +1,134 @@
+--- activate under main muc component
```

Comment:
These are instructions setting up the system as a whole, not just mod_fmuc. Maybe they should go to a separate md file?

Reply:
Yeah, while setting up the rest at the moment for running several on one machine I will create a md file with instructions and probably I can drop those then.